File: src/main/java/com/matchFit/follow/dto/response/GetMyFollowResponseDto.java
package com.matchFit.follow.dto.response;

import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class GetMyFollowResponseDto {
    private final Long postId;
    private final String title;
    private final String sports;
    private final String location;
    private final String date;
    private final int currentPeople;
    private final int maxPeople;
    private final String followedAt;
    private final int cost;
    private final String status;
}

File: src/main/kotlin/com/matchFit/follow/dto/response/GetMyFollowResponseDto.kt
package com.matchFit.follow.dto.response

data class GetMyFollowResponseDto(
    val postId: Long,
    val title: String,
    val sports: String,
    val location: String,
    val date: String,
    val currentPeople: Int,
    val maxPeople: Int,
    val followedAt: String?,
    val cost: Int,
    val status: String
)
